src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -69,4 +75,11 @@ object TryUtil extends LazyLogging {
       case f => f
     }
   }
+
+  def flatten[A](s: Seq[Try[A]]): Try[Seq[A]] = {
```

Comment:
This seems like the signature of `sequence` rather than `flatten`

http://www.scala-lang.org/files/archive/nightly/docs/library/index.html#scala.concurrent.Future$